### ISSUE TITLE
Payments - Credential Presentation & Verification

### ISSUE THREAD

**Issue by cyberphone** [openid/OpenID4VP#261]
Since 
https://github.com/digitallabor-berlin/eudiw-sca/blob/fbbaf948594998e57a21f772ed0b630798b32bff/openbanking-r2s.md
seems to be the currently only public payment solution building on OpenID4VP, this issue builds on this particular solution.

Apparently Payees (merchants) are supposed to verify received payment credentials.  To make this possible, payment credentials 
- must be sent in their entirety [*]
- must be signed

However, existing payment authorization system including EMV and W3C's SPC do neither depend on signed credentials, nor on verification by Payees.

Apart from the desire "shoehorning" payment authorizations into a framework originally designed for identity, what are the motives for having Payees verify payment credentials? Doesn't handing over personal data like account numbers to _untrusted parties_ like Merchants violate GDPR?  Wasn't the EUIDW once supposed to _enhance_ privacy?

Yes, #188 is a tall order but armed with information like above, this proposal is obviously not completely without merit. As the King once sang, "It's now or never".

Sincerely, Anders Rundgren

*] that OpenID4VP payment credentials do not support cool card images like Apple Pay, doesn't come as a surprise.  Payment rails aren't designed to handle potentially quite large image files.

**Comment by cyberphone** [comment]
An off-list commenter claimed that payment intermediaries may need account data.  For card payments this is true, for A2A payments it is not.

**Comment by cyberphone** [comment]
@Sakurann I'm closing this issue because it is obvious that the interest in payment authorization systems within the OpenID community is quite limited. For a possible (and IMO necessary) project leader, such an activity would be a full-time mission.

Well, it of course depends what the objectives are.  As Ronny Kahn of Nobid expressed early on LinkedIn, the solution doesn't have to be competitive since it will (through legal means) be FORCED upon Banks and Merchants.  When I questioned this stance, Ronny and the DigitalLabor team "unfriended" me on LinkedIn and blocked my access to their GitHub repository.  Some people are apparently very sensitive...

To make things worse, the DigitalLabor wallet has (as they rightfully claim) currently nothing to connect to, making the rest rather unimportant.  They rely on the Berlin Group "Signed Payment Request" which a very poor API extension framework.  Not even the organizations who pushed this proposal have made any implementations.  I was a part of this development effort but withdraw when I realized that it would never scale.  However, there is a need for something along these lines anyway.  [Open Banking 2.0](https://github.com/cyberphone/open-banking-2.0?tab=readme-ov-file#open-banking-20) is an attempt to address this in a less specific way.

@jogu
